### PR TITLE
Update to ipywidgets 6 and first lab extension

### PR DESCRIPTION
**Pull request by SylvainCorlay**
- Update to ipywidgets 6.0
 - First lab extension
 - Add LICENSE file
 - Remove jquery-resizable.

### CODE REVIEW

File: js/src/mpl.js [hunk 1]
```diff
@@ -160,7 +157,7 @@ mpl.figure.prototype._init_canvas = function() {
 
     var pass_mouse_events = true;
 
-    canvas_div.resizable({
+    /*canvas_div.resizable({
```
Comment (blink1073) [inline js/src/mpl.js]: Should this block be removed?

Comment (SylvainCorlay) [inline js/src/mpl.js]: Either removed, or I should put a WIP in the PR description and iterate to make this work properly.

Comment (SylvainCorlay) [inline js/src/mpl.js]: Btw, would it make sense to have a phosphor counterpart to `jquery.resizable` with a similar UI (handle in the corner) but also phosphor resize events?

Comment (blink1073) [inline js/src/mpl.js]: Maybe in the `@jupyterlab/domutils` package I'm working on... https://github.com/blink1073/domutils

Comment (blink1073) [inline js/src/mpl.js]: (which will be merged into the main repo once we figure out https://github.com/phosphorjs/phosphor/issues/213

**Comment by SylvainCorlay** [comment]
@blink1073 I will make another pass to remove jquery calls, replace buttons with actual jupyter buttons etc...

**Pull request merged**